client/package.json

```diff
@@ -189,7 +189,7 @@
 		"react-markdown": "^8.0.7",
 		"react-modal": "^3.16.1",
 		"react-redux": "^8.1.3",
-		"react-router-dom": "^6.10.0",
+		"react-router-dom": "^6.23.1",
```

Comment:
It is necessary due to this issue https://github.com/remix-run/react-router/pull/10768.
Version 6.10.0 doesn't support optional parameters (:lang?) on the useMatch hook.